sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLEnv.scala

```diff
@@ -55,7 +55,7 @@ private[hive] object SparkSQLEnv extends Logging {
           maybeKryoReferenceTracking.getOrElse("false"))
 
       sparkContext = new SparkContext(sparkConf)
-      sqlContext = SparkSession.withHiveSupport(sparkContext).wrapped
+      sqlContext = SparkSession.builder.enableHiveSupport().getOrCreate().wrapped
```

Comment:
it would be better if we get the context from the session:
```
val sparkSession = SparkSession.builder.conf(sparkConf).enableHiveSupport().getOrCreate()
sqlContext = sparkSession.wrapped
sparkContext = sparkSession.sparkContext
```